README.md

```diff
@@ -78,6 +79,7 @@ This library provides a full and reusable custom camera, which:
 ```java
          final Intent intent = new Intent(getActivity(), VideoCaptureActivity.class);
          intent.putExtra(VideoCaptureActivity.EXTRA_CAPTURE_CONFIGURATION, config);
+         intent.putExtra(VideoCaptureActivity.EXTRA_CAMERA_FACING, PredefinedCameraFacing.REAR_FACING); //Opens rearfacing camera by default.
```

Comment:
Would not expose this via an intent, but only keep it in the Capture configuration. -> Good apis only have one way to achieve a certain function and the camera option seems to fit best with the capture configuration.